src/books.js

```diff
@@ -86,15 +144,31 @@ const dictionary = {
     ],
 };
 
-function booksByAuthor() {
+function booksByAuthor(dictionaryObj) {
   // Your code here:
-  
+  const arrOfBooks = [];
+  for (let authorName in dictionaryObj ){
+    dictionaryObj[authorName].forEach( (x,y) =>{
+              const newBookObject ={};
+              newBookObject.title = dictionaryObj[authorName][y][0];
```

Comment:
Since you already defined `x`, you could just do
```suggestion
              newBookObject.title = x[0];
              newBookObject.pages = x[1];
```
🤣